WTWSMS/common/landed_titles/WtWSMS_tribes.txt

```diff
@@ -952,13 +952,19 @@ d_goths = {
 	# tribe = yes
 	
 	allow = {
-		always = no
+		OR = {
+			culture = gothic
+			culture = crimean_gothic
+			culture = ostrogothic
+		}
+		controls = 560
+		prestige = 500
```

Comment:
@loup99 

The point here was that Gothic characters can restore Gothic duchies (etc.). It's a shame that those Titular Titles die out so easily, so I wanted to put in a reasonable fix for it. Likewise, Angle cultured people should be able to restore Angloland, and so on.